### DISCUSSION TITLE
Hostinger custom domain not working?

### DISCUSSION THREAD

**Discussion by daffaromero** [vercel/community#3543] [Help] [answered]
### Summary

Hello! I tried changing the domain of my Vercel deployment to a custom domain I got through Hostinger. Keep in mind that this Hostinger domain originally points to WordPress site. 

I pointed it to my Vercel deployment by changing the nameservers, and now I can not access the domain outside of the apex domain pointed to my Vercel deployment. I can't access the wp-admin, the GraphQL API, nothing. The Vercel deployment works fine through the apex domain.

I'm pretty lost here, am I doing anything wrong? Should I just wait for the changes to be propagated?

### Example

https://visitsidoharjo.net

### Steps to Reproduce

1. Create a Vercel deployment (Next.js)
2. Create a Hostinger deployment (WordPress)
3. Obtain Hostinger-provided domain
4. Change Vercel domain to Hostinger-provided domain through changing nameservers in Hostinger's hPanel

**Comment by Dracpox** [comment, marked as answer]
Hi @daffaromero 🙂 When you configure a custom domain at Vercel (or elsewhere), this domain is pointing to whatever you have deployed in Vercel. In your case, I assume that you have your Wordpress deployed at Hostinger, right? Not at Vercel. That's why your apex domain configured in Vercel can't access the wp-adming panel, because is not there, it's in Hostinger. In short, t**his is not a problem with Vercel or Hostinger, it's a misunderstanding with domains** 😉 But don't worry, you can learn from this

**What can you do at this point?** Well, you can create a subdomain and copy there your Wordpress ([link to docs to do it](https://support.hostinger.com/en/articles/6601521-how-to-copy-a-wordpress-website-to-another-domain-name)). By doing so, you will be able to access your Wordpress wp-adming panel and GraphQL API through the subdomain and access to your Vercel deployment through your apex domain.

I hope this works for you 😊
